kzz7yh-a10381/cod-ve07v1_kzz7yh-a10381.xml

```diff
@@ -151,7 +153,7 @@
       <lb ed="#V" n="44"/>et ordinem eoipso quod finem habet. In re enim que habet 
       <lb ed="#V" n="45"/>finem opetet esse ordinem ad finem. Quia ergo modus creatu 
       <lb ed="#V" n="46"/>re hoc est eius limitatio vel mensura deum representat 
-      <lb ed="#V" n="47"/>sub ratione cause efficientis: et pulchritudo eius deum 
+      <lb ed="#V" n="47"/>sub rafrespondeotione cause efficientis: et pulchritudo eius deum 
```

Comment:
I'm not sure what is happening here. this seems like maybe an accidental change. In any case it was correct as is.
```suggestion
      <lb ed="#V" n="47"/>sub ratione cause efficientis: et pulchritudo eius deum 
```